motimo-domain/src/main/java/kr/co/domain/subGoal/SubGoal.java

```diff
@@ -25,8 +29,14 @@ private SubGoal(String title, int importance) {
         this.completed = false;
     }
 
-    public void changeCompleted() {
+    public void toggleCompleted() {
         completed = !completed;
         completedChangedAt = LocalDateTime.now();
     }
+
+    public void userChecked(UUID userId) {
```

Comment:
권한 체크를 수행하는 메서드인 것 같은데 userChecked는 동사로 보이기보다는 과거 시제처럼 읽혀서 이 메서드가 무엇을 하는지 직관적으로 파악하기 어려운거 같아요!

```suggestion
    public void validateOwner(UUID userId) {     // 예시
```

의도를 분명히 나타내는 네이밍으로 바꾸면 더 좋을 것 같아요!